silence/silence.go

```diff
@@ -276,6 +325,11 @@ func (s *Silences) setSilence(sil *pb.Silence) error {
 	if err != nil {
 		return err
 	}
+
+	_, err = s.mc.Add(sil)
+	if err != nil {
+		return err
+	}
```

Comment:
You you're saying, we skip initializing the cache all together and let it warm up while using?